stream.go

```diff
@@ -414,7 +423,7 @@ func (s *Stream) readData(hdr header, flags uint16, conn io.Reader) error {
 	}
 
 	// Decrement the receive window
-	atomic.AddUint32(&s.recvWindow, ^uint32(length-1))
+	s.recvWindow += ^uint32(length - 1)
```

Comment:
Whilst it is the same thing, this is a carry over from the atomic operation. We should change this to read

```
s.recvWindow -= length
```